python/vyos/logger.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2020 VyOS maintainers and contributors <maintainers@vyos.io>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this library.  If not, see <http://www.gnu.org/licenses/>.
+
+# import logging and related modules
+import logging
```

Comment:
Why import the complete class and one line lter only specific components?

Reply:
Not all standard logging handlers are inside the `logging` module, it includes only `NullHandler`, `StreamHandler` and `FileHandler`. `SysLogHandler` and `RotatingFileHandler` need to be imported from the `logging.handlers` module additionally.